### ISSUE TITLE
[OJS] Remove unused locales

### ISSUE THREAD

**Issue by mtub** [pkp/pkp-lib#2237]
There are (very old) locale folders that belonged to translations we no longer maintain and that are not mentioned in the registry. I suggest removing them. They have never been updated for OJS3, and removing them might reduce confusion. But be careful: among those is fr_FR, which probably people will want to re-create in the future, but I guess it will be better to then build upon fr_CA than to use a massively outdated fr_FR from OJS2.

* es_AR (we have es_ES)
* fr_FR (we have fr_CA)
* mk_MK
* no_NO (we have nb_NO)
* pl_PL
* sk_SK

If we don't remove them, we should maybe add them to the registry… either way, I'd like to see this cleaned up a bit. @asmecher , I think this should be decided before the next release so we can have a fresh start after the release…

**Comment by mtub** [comment]
There are a few other locales that are part of the registry but are from OJS 10 years ago. I'd be willing to even remove some more locales… Someone updating a locale that old will probably only look for missing translation, so errors and outdated language will get carried on. A fresh start would probably better for these locales, too.

**Comment by mtub** [comment]
Maybe @jmacgreg has something to say about this, too…

**Comment by asmecher** [comment]
@mtub, I would recommend removing them from github, and agree that now is a good time to do it. If someone really does want to build on an old translation, it can always be dredged up.

**Comment by jmacgreg** [comment]
Yep, these can go as far as I'm concerned.

**Comment by mtub** [comment]
Thank you... please note that I might have tricked you by adding a second proposal in my first comment ;) - so do you also agree to remove some more really incomplete and outdated locales?

**Comment by jmacgreg** [comment]
Yep!

**Comment by mtub** [comment]
Ok, I will prepare a pull request (only for OJS3 at the moment)

**Comment by mtub** [comment]
These are my additional suggestions for removal from OJS3

* fa_IR
* gl_ES
* ja_JP
* ml_IN
* vi_VN
* zh_TW

Those have to be remove from the registry as well. Can maybe someone take care of a patch?

**Comment by mtub** [comment]
* el_GR
* ro_RO

**Comment by mtub** [comment]
Regarding lib/pkp: I can take care of it, but it will need some thoughts about the other applications. I will check if any of the removed languages are needed by OMP's master branch. OCS is a little more complicated because of the somewhat messy situation with the branches in OCS. New PR coming in tomorrow...

**Comment by asmecher** [comment]
Don't worry about OCS -- just have a look at the OMP locale list, which OTOH shouldn't contain any cruft (I don't think we listed locales there that weren't up to date as of OMP 1.0, so it's not nearly as bad as old OJS 2.x locales).

**Comment by mtub** [comment]
Will remove all of the locales listed above, with the exception of `el_GR`, which is used by OMP.

That is:
- es_AR
- fa_IR
- fr_FR
- gl_ES
- ja_JP
- mk_MK
- ml_IN
- no_NO
- pl_PL
- ro_RO
- sk_SK
- vi_VN
- zh_TW

**Comment by asmecher** [comment]
Merged #2240 -- can this be closed, @mtub?

**Comment by mtub** [comment]
Yes :)
